geth-utils/build.rs

```diff
@@ -8,7 +8,11 @@ fn main() {
     gobuild::Build::new().file("./lib/lib.go").compile(lib_name);
 
     // Files the lib depends on that should recompile the lib
-    let dep_files = vec!["./gethutil/trace.go"];
+    let dep_files = vec![
+        "./gethutil/asm.go",
+        "./gethutil/trace.go",
+        "./gethutil/util.go",
```

Comment:
The lib currently doesn't depend on `asm.go` and `util.go`, but you're right that it's probably a good idea to add these here anyway because that could change at any time.